cli/src/main/java/org/infinispan/cli/commands/kubernetes/Create.java

```diff
@@ -49,22 +48,28 @@ public CommandResult exec(ContextAwareCommandInvocation invocation) {
    @CommandDefinition(name = "cluster", description = "Creates a cluster")
    public static class Cluster extends CliCommand {
 
-      @Option(shortName = 'n', description = "Select the namespace")
+      @Option(shortName = 'n', description = "Specifies the namespace where the cluster is created. Uses the default namespace if you do not specify one.")
       String namespace;
 
-      @Option(shortName = 'r', description = "The number of replicas", defaultValue = "1")
+      @Option(shortName = 'r', description = "Specifies the number of replicas. Defaults to 1.", defaultValue = "1")
       int replicas;
 
-      @Option(name = "expose-type", completer = ExposeCompleter.class)
+      @Option(name = "expose-type", completer = ExposeCompleter.class, description = "Makes the service available on the network through a LoadBalancer, NodePort, or Route.")
```

Comment:
"Expose the service externally available through a ..."?

Reply:
Another instance of trying to avoid using the same wording as the parameter in its description. This helps with translation and non-native speakers.